recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
```

Comment:
Any update on this? I was just going through my backlog of reviewed PRs and noticed that this PR was still outstanding.